hw_5/task_01_path.py

```diff
@@ -0,0 +1,13 @@
+"""
+Напишите функцию, которая принимает на вход строку - абсолютный путь до файла.
+Функция возвращает кортеж из трёх элементов: путь, имя файла, расширение файла.
+"""
+
+
+def path_file(path_file):
+    *path, name, extension = path_file.replace('.', "\\").split('\\')
+    return tuple(['\\'.join(path), name, extension])
```

Comment:
Если через запятую перечислять после `return` переменные, то они создадут кортеж при возврате. То есть явно определять его не нужно.
```suggestion
    return '\\'.join(path), name, extension
```